TeamCode/src/main/java/org/firstinspires/ftc/teamcode/teleop/StrafeDrive.java

```diff
@@ -2,18 +2,20 @@
 
 import com.qualcomm.robotcore.eventloop.opmode.OpMode;
 import com.qualcomm.robotcore.eventloop.opmode.TeleOp;
+import com.qualcomm.robotcore.hardware.DcMotor;
 import com.qualcomm.robotcore.util.ElapsedTime;
 import com.qualcomm.robotcore.util.Range;
 
-import org.firstinspires.ftc.teamcode.subsystems.HWC;
+import org.firstinspires.ftc.teamcode.subsystems.hardware.Hardware;
+import org.firstinspires.ftc.teamcode.subsystems.hardware.HardwareElement;
 
 /**
  * TeleOp OpMode for simply driving with strafing wheels
  */
 @TeleOp(name = "StrafeDriveBasic", group = "Iterative OpMode")
 public class StrafeDrive extends OpMode {
     private final ElapsedTime time = new ElapsedTime();
-    HWC robot; // Declare the object for HWC, will allow us to access all the motors declared there!
+    private Hardware robot;
 
     // init() Runs ONCE after the driver hits initialize
```

Comment:
I know this isn't your mistake, but please add periods to the ends of these comments.